include/item.h

```diff
@@ -550,6 +550,6 @@ T inclusive_scan_over_group(const group<Dims>& g, V x, BinaryOperation binary_op
 #endif
 }
 
-#endif // !WORKAROUND_COMPUTECPP
+#endif // !WORKAROUND_COMPUTECPP || WORKAROUND(COMPUTECPP, 2, 10)
```

Comment:
Why is there an end of `WORKAROUND(COMPUTECPP, 2, 10)` indicated here? I don't see a corresponding opening (just `2, 9`.

Reply:
Removed.